src/Modules/OrchardCore.Commerce/Endpoints/Services/ShoppingCartService.cs

```diff
@@ -64,11 +63,11 @@ public async Task<string> RemoveLineAsync(ShoppingCartLineUpdateModel line, stri
 
     public async Task<string> AddItemAsync(ShoppingCartLineUpdateModel line, string token, string shoppingCartId = null)
     {
-        string errored = string.Empty;
+        string strError = string.Empty;
```

Comment:
Please don't use Hungarian notation in C#, it's only for weakly typed languages. Also this variable is not necessary at all. You can return the values directly when they are available.